pkg/network/tracer/tracer_linux_test.go

```diff
@@ -456,7 +453,6 @@ func (s *TracerSuite) TestConntrackExpiration() {
 	// write newline so server connections will exit
 	_, err = c.Write([]byte("\n"))
```

Comment:
so we still need this if we're removing the assertion above?

Reply:
Yes, we need a clean exit from the server loop. The error in the job failure is caused by the server loop not exiting cleanly.